Core/include/Acts/Geometry/Extent.hpp

```diff
@@ -68,7 +68,7 @@ struct Extent {
   /// Extend with another extent
   /// @param other is the source Extent
   void extend(const Extent& other) {
-    for (auto ir = 0; ir < other.ranges.size(); ++ir) {
```

Comment:
Ha, that was magically bound to an int?

Reply:
Well, not magically. Since a `0` literal has `int` as its type, `auto ir = 0` must be `int` as well.